docs/ref/contrib/admin/index.txt

```diff
@@ -2507,11 +2507,32 @@ Templates can override or extend base admin templates as described in
     * ``site_title``: :attr:`AdminSite.site_title`
     * ``site_url``: :attr:`AdminSite.site_url`
     * ``has_permission``: :meth:`AdminSite.has_permission`
+    * ``available_apps``: a list of applications from the
+      :doc:`application registry </ref/applications/>` available for the
+      current session. Each entry in the list is a dict representing an application
+      with the following keys:
+
+      * ``app_label``: the application label
+      * ``app_url``: the url of the ``application index`` in the admin
```

Comment:
url -> URL
I'd omit the backticks around application index since it's not a code thing